docs/charts/index.yaml

```diff
@@ -1,9 +1,36 @@
 apiVersion: v1
 entries:
   camel-k:
+  - apiVersion: v2
+    appVersion: 2.1.0-SNAPSHOT
```

Comment:
I don't think we should publish a snapshot. We should recreate the helm chart for 2.0.0 and push it again with the role fixed.

Reply:
Sure, makes sense, thanks for spotting it.